client/ng/components/live-chat/live-chat.component.js

```diff
@@ -0,0 +1,50 @@
+// jshint esversion: 6
+
+import module from './live-chat.module';
+import template from './live-chat.html';
+
+class LiveChat {
+  constructor() {
+    'ngInject';
+    this.messages = [
```

Comment:
wrzucić to do this.$onInit
(https://toddmotto.com/angular-1-5-lifecycle-hooks#oninit)